save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/filters/ExecutionFilters.kt

```diff
@@ -0,0 +1,20 @@
+package com.saveourtool.save.filters
+
+import kotlinx.serialization.Serializable
+
+/**
+ * Aff filters in one property
+ * @property startTime to filter by [startTime] execution formatting: <year, month, day>
+ * @property endTime to use filter by [endTime] execution formatting: <year, month, day>
+ */
+@Serializable
+data class ExecutionFilters(
+
+    val startTime: Triple<Int, Int, Int>,
```

Comment:
Please create a data class instead of `Triple`. Tuples with unnamed fields work well only when they represent local variables; with `Triple` as a class property it will become harder to keep track if all fields are unpacked correctly among the code and also harder to extend these fields, should there be a neccessity.

Also, there is `kotlinx.datetime.LocalDateTime` that can be used in both JS and JVM. Why have you decided not to use it?

Reply:
Fixed